cloudbuild-e2e-cloud-run.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 Google LLC
```

Comment:
Delete this one for now, cloud run requires the "push" mode which isn't implemented yet for Go

Reply:
Done